### DISCUSSION TITLE
Target class [Spatie\Honeypot\HoneypotViewComposer] does not exist.

### DISCUSSION THREAD

**Discussion by jasperf** [spatie/laravel-honeypot#72] [Q&A] [answered]
We just upgraded to the latest Laravel 8.18.1 version and also updated many composer packages including Laravel Honeypot (3.0.1). Now we however get the error

```
Illuminate\Contracts\Container\BindingResolutionException
Target class [Spatie\Honeypot\HoneypotViewComposer] does not exist.
```

and yes, the View Composer file is not there under `app/Http/ViewComposers` nor has it been registered. Any idea why this did not occur during the upgrade? Did we miss a step perhaps?

**Comment by freekmurze** [comment]
Did you already try clearing the view cache?

**Comment by jasperf** [comment]
Did do a `php artisan view:clear` but still the same issue. I will investigate some more this morning.

**Comment by jasperf** [comment, marked as answer]
We needed to adjust the path in one controller

```php
- \View::composer("{$domainView}::index", \Spatie\Honeypot\HoneypotViewComposer::class);
+ \View::composer("{$domainView}::index", \Spatie\Honeypot\View\HoneypotViewComposer::class);
```

Now all good. Thanks for your feedback @freekmurze .
